project/src/components/favorites/favorites.tsx

```diff
@@ -10,9 +11,9 @@ function Favorites(): JSX.Element {
             <li className="favorites__locations-items">
               <div className="favorites__locations locations locations--current">
                 <div className="locations__item">
-                  <a className="locations__item-link" href="#">
+                  <Link className="locations__item-link" to="/">
```

Comment:
```suggestion
                  <Link className="locations__item-link" to={AppRoute.MAIN}>
```
тут тоже используй енам лучше (нужно исправить везде будет)